src/scripts/blocks/post-list/DisplayComponent.js

```diff
@@ -105,6 +109,21 @@ class DisplayComponent extends Component {
               checked={ attributes.categoryRelated }
               onChange={ this.createUpdateAttribute('categoryRelated') }
             />}
+            { attributes.type === 'select' && <ToggleControl
+              label={ __('Display excerpt', 'benenson') }
+              checked={ attributes.displayExcerpt }
+              onChange={ this.createUpdateAttribute('displayExcerpt') }
+            />}
+            { attributes.type === 'select' && (<div>
+            <label htmlFor="editorButtonText">{ __('Button Text:', 'benenson') } </label>
+             <RichText
+              name={ 'editorButtonText' }
+              className={ ['editorButtonText'] }
+              placeholder={ __('(CTA Text)', 'benenson') }
+              value={ attributes.ctaText }
+              keepPlaceholderOnFocus={ true }
+              onChange={ this.createUpdateAttribute('ctaText') }
+            ></RichText></div>)}
```

Comment:
`RichText` can be self-closed: `/>` rather than `></RichText>`.
Missing space before closing curly brace.